saleor/graphql/page/mutations/attributes.py

```diff
@@ -86,10 +87,10 @@ def clean_attributes(
             errors["attribute_ids"].append(error)
 
     @classmethod
-    def perform_mutation(cls, _root, info, **data):
-        errors = defaultdict(list)
-        page_type_id: str = data["page_type_id"]
-        attribute_ids = data["attribute_ids"]
+    def perform_mutation(  # type: ignore[override]
+        cls, _root, info: ResolveInfo, /, *, attribute_ids, page_type_id
+    ):
+        errors: defaultdict[str, List[ValidationError]] = defaultdict(list)
 
         # retrieve the requested page type
         page_type = cls.get_node_or_error(
```

Comment:
Unsure if that's voluntary: it lost the type hint
```suggestion
        page_type: str = cls.get_node_or_error(
```

Reply:
Not sure I follow. What exactly was lost? `get_node_or_error` is a generic function with its return type based on the type passed as `only_type`.